mozilla-release/browser/app/profile/firefox.js

```diff
@@ -1272,7 +1272,8 @@ pref("services.sync.prefs.sync.browser.safebrowsing.malware.enabled", true);
 pref("services.sync.prefs.sync.browser.search.update", true);
 pref("services.sync.prefs.sync.browser.sessionstore.restore_on_demand", true);
 pref("services.sync.prefs.sync.browser.startup.homepage", true);
-pref("services.sync.prefs.sync.browser.startup.page", true);
+pref("services.sync.prefs.sync.browser.startup.restoreTabs", true);
+pref("services.sync.prefs.sync.browser.startup.addFreshTab", true);
```

Comment:
@maxim-cliqz we use that pref in the extension and it seems a bit weird to add 2 new prefs for what we want to do. What about adding a new value in browser.startup.page? eg: 4 :) here is the list with all the values - http://kb.mozillazine.org/Browser.startup.page